apis/core/v1alpha1/restoresession_types.go

```diff
@@ -247,6 +256,15 @@ type KubeDBManifestOptions struct {
 	// +optional
 	AuthSecretName string `json:"authSecretName,omitempty"`
 
+	// Archiver specifies whether to restore the Archiver manifest or not
+	// +kubebuilder:default=true
+	// +optional
+	Archiver *bool `json:"archiver,omitempty"`
+
+	// ArchiverRef specifies the name of the ArchiverRef
+	// +optional
+	ArchiverRef *kmapi.ObjectReference `json:"archiverRef,omitempty"`
```

Comment:
// ArchiverRef specifies the new name and namespace of the Archiver yaml after restore